netbout-web/src/main/java/com/netbout/rest/MarkdownTxtmark.java

```diff
@@ -110,4 +161,19 @@ private static String makeLineBreak(final String txt) {
         matcher.appendTail(result);
         return result.toString();
     }
+
+    /**
+     * Fix code blocks marked with "```" as they are incorrect processed.
+     * @param txt Text to parse
+     * @return Fixed text with correct code blocks.
+     */
+    private static String fixedCodeBlocks(final String txt) {
+        return txt
+            .replace("<code>`\r\n", MarkdownTxtmark.PRE_CODE)
```

Comment:
@krzyk Missed that. And using regexp will make unnecessary using of variables or constants for html strings.